django-stubs/core/management/commands/runserver.pyi

```diff
@@ -1,3 +1,12 @@
+from typing import Tuple
+
 from django.core.management.base import BaseCommand
 
-class Command(BaseCommand): ...
+class Command(BaseCommand):
+    requires_system_checks: bool
+    stealth_options: Tuple[str]
```

Comment:
Should be `Tuple[str, ...]` to have the same type as other commands. For example: https://github.com/django/django/blob/eac9ab7ebb1ce0cbbc79c4cf65e8f70b0635a240/django/core/management/commands/flush.py#L15

Reply:
Should this definition go into `BaseCommand` then?

Reply:
Yes, I think so: https://github.com/django/django/blob/eac9ab7ebb1ce0cbbc79c4cf65e8f70b0635a240/django/core/management/base.py#L212-L226

Please, check other props as well.

Reply:
I've updated the type for `stealth_options` and `base_stealth_options` in `BaseCommand` as they were declared as `Any`.